tests/test-rest-tags-controller.php

```diff
@@ -224,6 +224,45 @@ public function test_get_items_post_args() {
 		$this->assertEquals( 'DC', $data[0]['name'] );
 	}
 
+	public function test_get_terms_post_args_paging() {
+		$post_id = $this->factory->post->create();
+		$tag_ids = array();
+
+		for ( $i = 0; $i < 30; $i++ ) {
```

Comment:
@kjbenk In the future, you might want to do this paging test with a smaller set of sample data. Generally, with unit tests, you should use as little data as you actually need for the test, because generating new data incurs database writes, and those database writes add a not-inconsequential amount of time to a full run of the test suite.